data/korath/korath weapons.txt

```diff
@@ -577,6 +737,163 @@ effect "slicer impact"
 	"random velocity" .1
 	"velocity scale" -.002
 
+
+outfit "Digger Mining Turret"
+	category "Turrets"
+	cost 509000
+	thumbnail "outfit/korath digger turret"
+	"mass" 14
+	"outfit space" -16
+	"weapon capacity" -16
+	"asteroid scan power" 15
+	"turret mounts" -1
+	"required crew" 1
+	weapon
+		sprite "projectile/digger"
+			"frame rate" 20
+		"hardpoint sprite" "hardpoint/digger turret"
+		sound "korath digger"
+		"hit effect" "korath digger hit"
+		"inaccuracy" 1.5
+		"turret turn" 4.6
+		"velocity" 480
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 2.9
+		"firing heat" 4.9
+		"shield damage" 0.8
+		"hull damage" 1.3
+		"minable damage" 5.2
+		"disruption damage" .08
+	description `When the Korath Exiles needed a cheap mining laser for their large vessels, they re-engineered this ancient but robust design. The mild shield disruption effect makes it somewhat useful in combat if required.`
+
+outfit "Digger Mining Beam"
+	category "Guns"
+	cost 419000
+	thumbnail "outfit/korath digger"
+	"mass" 10
+	"outfit space" -10
+	"weapon capacity" -10
+	"asteroid scan power" 10
+	"gun ports" -1
+	weapon
+		sprite "projectile/digger thin"
+			"frame rate" 20
+		sound "korath digger"
+		"hit effect" "korath digger hit"
+		"inaccuracy" 0.1
+		"velocity" 480
+		"lifetime" 1
+		"reload" 1
+		"firing energy"  2.1
+		"firing heat" 4.3
+		"shield damage" 0.5
+		"hull damage" 0.9
+		"minable damage" 4.6
+		"disruption damage" .06
+	description `The gun version of the classic Korath mining laser allows a larger mothership to release a cloud of fighter-sized miners to harvest resources when working faster-moving or more dangerous asteroid fields.`
+
+effect "korath digger hit"
+	sprite "effect/korath digger hit"
+		"no repeat"
+		"frame rate" 30
+	"lifetime" 5
+	"random angle" 30
+	"velocity scale" .005
+
+
+outfit "Langrage Hyper-Heaver"
+	category "Secondary Weapons"
+	cost 3450000
+	thumbnail "outfit/korath heaver"
+	"mass" 46
+	"outfit space" -46
+	"weapon capacity" -46
+	"fuel capacity" 200
+	"turret mounts" -1
+	"required crew" 1
+	weapon
+		sprite "projectile/heaver hyperspace"
+			"frame rate" 12
+			rewind
+		"hardpoint sprite" "hardpoint/korath heaver"
+		icon "icon/korath heaver"
+		velocity 30
+		lifetime 8
+		"hit force" 3600
+		"hit effect" "heaver hyperspace hit"
+		sound "heaver"
+		submunition "Heaver Shard" 20
+		"turret turn" 2.2
+		"reload" 40
+		"burst reload" 12
+		"burst count" 6
+		"firing energy" 31
+		"firing heat" 190
+		"firing fuel" 12.5
+		"inaccuracy" 0.5
+	description `Langrage Hyper-Heavers raise the use of improvised ammunition to a fine art. A small Banisher propels scrap metal, nearby asteroids, compressed garbage, or other debris into an unstable hyperspace window.`
+	description `At close range, the jumping langrage merely forces the enemy back. Further along, being forced to hyper transforms it into a rapidly-expanding cloud of deadly projectiles as it returns to normal space.`
```

Comment:
Be aware that parent projectiles inherit the damage of their submunitions. This description makes it sound to me like the weapon isn't working as intended; the first stage of the projectile won't only deal hit force, and the Shards will include the damage of the Splinters, meaning that I believe that this weapon is dealing double the damage that it's intended to, making it one of the strongest weapons in the game.
![image](https://user-images.githubusercontent.com/17688683/180982905-5a47b1f7-a930-42af-a4cf-e17aa7bc3044.png)

If you want to have this weapon work as intended:
* The Shard should deal no damage and have no hit force, as a Shard's damage will be contained within its Splinters.
* The main weapon should deal negative damage equal to the damage of all Splinters so that it cancels them out.

And please use a damage value for the Splinter that shouldn't be a repeating digit. 5.7 damage per shard will result in 342 damage per shot (whereas I believe the intended damage per shot is 340).